.github/workflows/build-test-e2e.yml

```diff
@@ -99,10 +122,34 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: Install Playwright Browsers
         run: yarn playwright install --with-deps
-      - name: Confirm Docker container is running
+      - name: Check for common errors
         run: |
+          if [ ! -e "/$(pwd)/${{ env.BUILD_DIR }}/${{ env.BUILD_NAME }}" ]; then
+            echo "::error:: downloaded WAR not found at expected location"
+            exit 1
+          fi
           if [[ $(docker ps --filter "name=${{ env.CONTAINER_NAME }}" --format '{{.Names}}') != "${{ env.CONTAINER_NAME }}" ]]; then
-            echo "::error SWC docker container is not running"
+            echo "::error:: SWC docker container is not running"
+            exit 1
+          fi
+          if ! docker port "${{ env.CONTAINER_NAME }}" | grep -q "8080/tcp -> 0.0.0.0:8888"; then
+              echo "::error:: SWC container configuration does not specify expected port mapping"
+              exit 1
+          fi
+          if [[ "$(docker exec "${{ env.CONTAINER_NAME }}" bash -c "curl -sS -o /dev/null -w '%{http_code}\n' http://127.0.0.1:8080")" != "200" ]]; then
+            echo "::error:: SWC container port not accessible from container"
+            exit 1
+          fi
+          if [[ "$(docker exec "${{ env.CONTAINER_NAME }}" bash -c "curl -sS -o /dev/null -w '%{http_code}\n' http://172.17.0.1:8888")" != "200" ]]; then
+            echo "::error:: SWC host port not accessible from container"
+            exit 1
+          fi
```

Comment:
Are these the same?

Reply:
They're a bit different -- the first attempts to connect to port 8080 on the local loopback address of the container itself - `http://127.0.0.1:8080`. The second attempts to connect to port 8888 on the Docker host from within the container - `http://172.17.0.1:8888`. If both return a 200, then SWC is accessible at both expected ports from within the container.